main.nf

```diff
@@ -24,55 +24,77 @@ def helpMessage() {
       -profile                      Configuration profile to use. Can use multiple (comma separated)
                                     Available: conda, docker, singularity, awsbatch, test and more.
 
-    Options:
-      --genome                      Name of iGenomes reference
-      --singleEnd                   Specifies that the input is single end reads
-    Strandedness:
-      --forward_stranded            The library is forward stranded
-      --reverse_stranded            The library is reverse stranded
-      --unstranded                  The default behaviour
+    Generic:
+      --singleEnd                   Specifies that the input is single-end reads
 
-    References                      If not specified in the configuration file or you wish to overwrite any of the references.
+    References:                     If not specified in the configuration file or you wish to overwrite any of the references.
+      --genome                      Name of iGenomes reference
       --star_index                  Path to STAR index
       --hisat2_index                Path to HiSAT2 index
-      --fasta                       Path to Fasta reference
+      --salmon_index                Path to Salmon index
+      --fasta                       Path to genome fasta file
+      --transcript_fasta            Path to transcript fasta file
+      --splicesites                 Path to splice sites file for building HiSat2 index
       --gtf                         Path to GTF file
       --gff                         Path to GFF3 file
       --bed12                       Path to bed12 file
-      --saveReference               Save the generated reference files the the Results directory.
-      --saveTrimmed                 Save trimmed FastQ file intermediates
-      --saveAlignedIntermediates    Save the BAM files from the Aligment step  - not done by default
+      --saveReference               Save the generated reference files to the results directory
```

Comment:
Is this params similar to `saveGenomeIndex` from nf-core/chipseq?
I guess yes, so we should update both chipseq and sarek to use `saveReference` instead of `saveGenomeIndex`.
We should pay more attention to params that can be shared within pipelines...

Reply:
I opened an issue for that in nf-core/tools as we should probably update that in the template? https://github.com/nf-core/tools/issues/417

Reply:
it not something that is needed by all pipeline I guess, I would more add that in guidelines for developers...
(adding to the list)